js/kontent.js

```diff
@@ -0,0 +1,63 @@
+const projectId = "99b0b3b0-4838-0051-3d57-8af72f55e8a0";
+const apiBlogItems = `https://deliver.kontent.ai/${projectId}/items`;
+const apiBlogItem = `https://deliver.kontent.ai/${projectId}/items/`;
+const apiTaxonomies = `https://deliver.kontent.ai/${projectId}/taxonomies/`;
+const pageSize = 10;
+
+class URLBuilder {
+    constructor(baseURL) {
+        this.singleEndpoint = null;
```

Comment:
what is the meaning of `singleEndpoint` ? As someone reading this code I'm not sure what I would set this too or how it relates to the `baseURL`. Is it just a way to add things on to the `baseURL` ?

Reply:
It is for the taxonomies, and [for future use of] fetching a single post item:

```
export function buildBlogItemUrl(itemCodeName) {
    itemFetchKontentApi.singleEndpoint = itemCodeName;
    return itemFetchKontentApi.build();
}

export function buildTaxonomiesUrl() {
    taxonomiesFetchKontentApi.singleEndpoint = "blog_tags";
    return taxonomiesFetchKontentApi.build();
}
```

Reply:
Can rename it, if it is not clear

Reply:
It may be clear I just don't understand the what is being described. In what way is "blog_tags" a singleEndpoint when "items" is not?